ldk/csharp/OliveHelpsLDK/OliveHelpsLDK/Whispers/FormInputs.cs

```diff
@@ -217,20 +205,30 @@ public WhisperFormInput.Types.Select ToProto()
             return select;
         }
 
+        /// <summary>
+        /// The list of options the user can choose from.
+        /// </summary>
         public string[] Options { get; set; }
     }
 
-    public struct Telephone : ITelephone
+    /// <summary>
+    /// Presents an input for entering telephone numbers. You can both provide the prepopulated Value,
+    /// as well as providing a <see cref="Pattern"/> that the input must satisfy.
+    /// </summary>
+    public class Telephone : BaseInput
     {
-        public string Label { get; set; }
-        public string Tooltip { get; set; }
-        public int Order { get; set; }
-
+        /// <summary>
+        /// Initial value.
+        /// </summary>
         public string Value { get; set; }
 
+        /// <summary>
+        /// The pattern of the field to be populated.
+        /// TODO: Provide example.
```

Comment:
Same as other TODO

Reply:
These values are formatted as per HTML Input elements so linking to MDN